src/routes/Main/Main.tsx

```diff
@@ -22,11 +29,36 @@ export default function Main() {
 
   return (
     <div className="relative">
-      <div>
+      <div className="px-5">
+        <Title text="Search Characters" />
         <SearchForm initialSearch={searchText} onClick={handleSearch} />
+        <div className="flex items-center justify-center pt-10">
+          <Button onClick={() => refetch()}>Refresh</Button>
```

Comment:
```suggestion
          <Button onClick={refetch}>Refresh ↻</Button>
```
You don't have to create a new wrapper function each time, but pass a reference if the signatures match, for example here "() => void" for the wrapper and the original